src/main/java/racingcar/Cars.java

```diff
@@ -0,0 +1,27 @@
+package racingcar;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Cars {
+	private List<Car> cars;
+
+	public Cars(int carCount) {
+		this.cars = new ArrayList<>();
+		for(int i = 0; i < carCount; i++) {
+			cars.add(new Car());
+		}
+	}
+
+	public void moveCars() {
+		int moveResult;
+		for (Car car : this.cars) {
+			moveResult = car.tryMove(RandomNumberGenerator.createRandomNumber());
```

Comment:
이 부분을 테스트하려면 어떻게 해야될까요?
랜덤으로 값을 생성하는 객체는 운영에만 적용하고 테스트에서는 랜덤이 아닌 고정된 값을 생성하는 객체를 만들면 테스트 가능합니다
NumberGenerator 라는 인터페이스를 추가해서 랜덤으로 값을 생성하는 클래스와 고정된 값을 생성하는 클래스를 만들어보세요
고정된 값을 생성하는 클래스를 테스트 코드에서 사용해서 게임룰에 따라 자동차가 이동 했는지 체크해보시면 됩니다

```suggestion
	private List<Car> cars;
	private final NumberGenerator numberGenerator;
...
	public Cars(int carCount, NumberGenerator numberGenerator) {
			this.numberGenerator = new RandomNumberGenerator();
			this.cars = new ArrayList<>();
...			
		for (Car car : this.cars) {
			moveResult = car.tryMove(numberGenerator.createNumber());
```